tests/fixtures/schema/0.10.3/image-fixture-good.json

```diff
@@ -47,6 +47,10 @@
     ]
   },
   "image_type": "photograph",
+  "focal_point": {
+    "x": 460,
+    "y": 220
+  },
   "additional_properties": {
```

Comment:
Can you add 1 or 2 failing focal point cases? E.g. x or y with a string, or a field name other than x or y

And invoke the test here: https://github.com/washingtonpost/ans-schema/blob/master/tests/schema-tests-010.js#L820